assets/src/components/ladder.tsx

```diff
@@ -203,7 +225,7 @@ const timepointStatusYFromTimepoints = (
       )
     }
   }
-  return null
+  return 0
```

Comment:
The `|| -10` later means that it doesn't matter whether we fail with `null` or `0`. Did you mean to remove that now that this function won't return `null`?
(I think there's a related bug where trains that are legitimately at `0` get moved to `-10`)

Reply:
That makes it sound like returning `0` and removing the `|| -10` is the better thing to do?

Reply:
Made that change